lib/components/customs/CustomPicture.vue

```diff
@@ -137,11 +138,15 @@ export default {
 
 function doPreloadFallback ({ srcset, sizes }, crossorigin, callback = () => {}) {
   if (!process.server) {
-    const img = new global.Image();
-    img.onload = callback;
-    img.crossorigin = crossorigin;
-    img.sizes = sizes;
-    img.srcset = srcset;
+    if (isPictureSupported()) {
+      const img = new global.Image();
+      img.sizes = sizes;
+      img.srcset = srcset;
+      img.crossorigin = crossorigin;
+      img.onload = callback;
+    } else {
+      callback && callback();
```

Comment:
you dont need to proof the availability of the callback function. you defined a fallback in the function header